packages/zoltan/test/TestMPI/canarySelfMessages.c

```diff
@@ -20,42 +20,43 @@ int main(int narg, char**arg)
 
   int nrecvs = nprocs;
   int nsends = nprocs;
+  int i;
```

Comment:
There's nothing _wrong_ with this change, but there was nothing wrong with declaring `int i` inside each `for` loop either :-) and the latter is the preferred form in modern C and C++.